src/modal/component/Modal/Modal.styles.ts

```diff
@@ -80,90 +84,90 @@ box-sizing: content-box;
 }
 
 .Modal-option-list li.selected-wallet {
-    background: #ededed;
+  background: #ededed;
 }
 .Modal-option-list li .selected-wallet-text {
-    text-align: center;
+  text-align: center;
 }
 
-.Modal-option-list li .selected-wallet-text span{
-    font-size: 14px;
-    font-weight: 500;
+.Modal-option-list li .selected-wallet-text span  {
+  font-size: 14px;
+  font-weight: 500;
 }
 
-.derivation-paths {
-}
 .derivation-paths-list {
-    display: flex;
-    justify-content: center;
-    flex-direction: column;
-    align-items: center;
-    margin-bottom: 1rem;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 16px;
 }
+
 .Modal-body button, .Modal-body input {
-    background: inherit;
-    font-size: 0.889em;
-    font-family: inherit;
-    border-width: 1px;
-    border-style: solid;
-    border-color: inherit;
-    border-radius: 40px;
-    margin-top: 0.5rem;
-    padding: 0.55em 1.4em;
-    text-align: center;
-    color: inherit;
-    font-family: inherit;
-    transition: background 150ms ease-in-out;
-    line-height: 1.15;
-    cursor: pointer;
- }
+  background: inherit;
+  font-size: 14.2px;
```

Comment:
What's the reason for moving away from em?

Reply:
I moved to "px" from "em" because when we integrated the wallet-selector in the guest book example, the font size was affected and the UI would look different meaning in the guest book integration the font size  for buttons was too large and it did not look nice.